17-using-javascript.Rmd

```diff
@@ -0,0 +1,406 @@
+library(emojifonts)
```

Comment:
Did you intend for this to execute? If so, be sure to add it to the DESCRIPTION file in Imports.

Reply:
You are correct. I was doing some research on using Emojis in RMarkdown. The use of the library (I believe) allows to call out the Rocket emojicon.  Removing the line of text, it still works because the `r emo::ji("rocket")` does the same thing.